ejb/src/main/java/org/wildfly/httpclient/ejb/HttpEJBReceiver.java

```diff
@@ -482,6 +493,49 @@ public void discardResult() {
 
     private static class EjbContextData {
         final Set<Method> asyncMethods = Collections.newSetFromMap(new ConcurrentHashMap<>());
+    }
+
+    /*
+     * This class manages the relationship between the proxy's strong and weak affinity and
+     * the stickiness requirements of session beans resulting from session creation.
+     */
+    private class SessionCreationStickinessHandler implements HttpTargetContext.HttpStickinessHandler {
+        private final EJBReceiverSessionCreationContext sessionCreationContext;
```

Comment:
what is the point of receiving the EJBReceiverSessionCreationContext here, if it is not being used?